VultisigApp/VultisigApp/Services/BlockChainService.swift

```diff
@@ -121,13 +121,13 @@ final class BlockChainService {
         case .ethereum, .avalanche, .bscChain, .arbitrum, .base, .optimism, .polygon, .blast, .cronosChain:
             let service = try EvmServiceFactory.getService(forChain: coin.chain)
             let (gasPrice, priorityFee, nonce) = try await service.getGasInfo(fromAddress: coin.address)
-            let gasLimit = normalizeGasLimit(coin: coin, action: action)
+            let gasLimit = gasLimit ?? normalizeGasLimit(coin: coin, action: action)
```

Comment:
Since `gasLimit` it is just a limit , if the transaction spend less then the `gasLimit` the remainder will be returned, thus let's do this
the `gasLimit` we got from `estimate_gas`, and our default value , whichever is larger , let's use the larger one